cmd/metrics-server/app/options/options.go

```diff
@@ -81,6 +98,7 @@ func (o *Options) Flags(cmd *cobra.Command) {
 
 	flags.MarkDeprecated("deprecated-kubelet-completely-insecure", "This is rarely the right option, since it leaves kubelet communication completely insecure.  If you encounter auth errors, make sure you've enabled token webhook auth on the Kubelet, and if you're in a test cluster with self-signed Kubelet certificates, consider using kubelet-insecure-tls instead.")
 
+	o.Validate()
```

Comment:
This is bad place to call Validate, please call it in runCommand https://github.com/kubernetes-sigs/metrics-server/blob/master/cmd/metrics-server/app/start.go#L47